README.md

```diff
@@ -1 +1,57 @@
-# Mobile-Protfolio
\ No newline at end of file
+# Mobile-Protfolio
+
+This site will be used as a mobile application to show off Rizwan Hasan' protfolio, here he will display all projects that he worked on previously to prove his programming skills.
+______________
+# Built With <br>
+HTML and CSS <br>
+HTML and CSS linters - stylelint, webhint, lighthouse- <br>
+Visual Studio and GitHub <br>
+GITHub Page <br>
+Figma Template <br>
+_________________
+# Getting Started <br>
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites - <br>
+Vist the git hub page Github Page Link.
+
+Install - <br>
+If not already downloaded, download Visual Studio, atom or any other IDE source code editor It may also help to download GitHub Desktop application
+
+Setup - <br>
+If Github desktop application is insalled in your device you can download the prokect directly to that. Once downloaded GitHub will give you the option to create a clone of the project, this will then allow you to open said project to your IDE. <br>
+
+If you do not have the GitHub desktop application you can download the project as a zip file directly to you computer. when the download is complete extract the folder to a location of your choice. After this you can open you IDE of choice and open the project folder.
+
+
+Run tests <br>
+test can me ran locally on your meachine by using the run function on your IDE.
+
+Deployment <br>
+restrict deployment 
+_________________
+
+# 👤 Author1 <br>
+
+GitHub: @rizwan-learn <br>
+Twitter: @rizwanhasan21 <br>
+LinkedIn: rizwanhasan21 <br>
+
+_________________
+
+# 🤝 Contributing <br>
+Contributions, issues, and feature requests are welcome! <br>
+
+Feel free to check the https://github.com/Rizwan-learn/Hello_Microverse_Project to get the project.
+
+# Show your support <br>
+Give a ⭐️ if you like this project!
+___________________
+# Acknowledgments <br>
+Microverse <br>
+Figma <br>
+
+_________________
+
+# 📝 License <br>
+This project is MIT licensed.
```

Comment:
- [Optional] Add Live Demo link, you can use Github Pages to deploy your website in the cloud for free.